tests/test_api.py

```diff
@@ -6,7 +6,13 @@
 import os
 import pandas as pd
 import re
+from math import ceil
+import logging
```

Comment:
Based on what I read [here](https://peps.python.org/pep-0008/#imports), standard library imports should be on top. So the order of the imports in our case should be: 
```
import os
import re
from math import ceil
import logging
from time import perf_counter
import pandas as pd # third party library import
from schematic.schemas.generator import SchemaGenerator #Local application/library specific imports.
```
Could you also update that?